foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/queries/DatabaseQueryMechanism.java

```diff
@@ -215,7 +219,14 @@ public boolean isJPQLCallQueryMechanism() {
      * This should be overridden by subclasses.
      * @return the row count.
```

Comment:
I'd like extend comment there with additional possible return type.

Reply:
Sure. I'll add a comment to `org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeNoSelect()`, rather than the abstract class `DatabaseQueryMechanism`. Does that work?